tasklist/client/src/modules/bpmn-js/styled.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+const bpmnRendererColors = {
+  outline: {
+    fill: 'var(--cds-highlight)',
+  },
+  defaultFillColor: 'var(--cds-layer-01)',
+  defaultStrokeColor: 'var(--cds-icon-secondary)',
+  element: {
+    text: 'var(--cds-text-primary)',
+    background: {
+      default: 'var(--cds-layer-01)',
```

Comment:
Does this have any reason for using the specific layer color? If not let's just use `--cds-layer`

Reply:
This is copied directly from operate

Reply:
Let's fix it and use `--cds-layer` then